src/celeritas/optical/ScintillationData.hh

```diff
@@ -29,67 +35,151 @@ struct ScintillationComponent
     //! Whether all data are assigned and valid
     explicit CELER_FUNCTION operator bool() const
     {
-        return yield_prob > 0 && lambda_mean > 0 && lambda_sigma > 0
-               && rise_time >= 0 && fall_time > 0;
+        return yield_frac > 0 && yield_frac <= 1 && lambda_mean > 0
+               && lambda_sigma > 0 && rise_time >= 0 && fall_time > 0;
     }
 };
 
 //---------------------------------------------------------------------------//
 /*!
- * Data characterizing the scintillation spectrum.
+ * Data characterizing material-only scintillation spectrum information.
  *
  * \c yield is the characteristic light yield of the material.
  * \c resolution_scale scales the standard deviation of the distribution of the
  * number of photons generated.
+ * \c components stores the fast/slow/etc scintillation components for this
+ * material.
  */
-struct ScintillationSpectrum
+struct MaterialScintillationSpectrum
 {
     real_type yield{};
```

Comment:
Maybe we should name this `yield_per_energy` and  definitely should note that it has units of 1/MeV. It looks like an average number of particles otherwise.